get_dir.c

```diff
@@ -0,0 +1,38 @@
+#include "shell.h"
+#include <dirent.h>
+
+char **get_dir(char *path)
```

Comment:
`path` is the token that matches the `command_path` string. In this case, `/bin`. The value was found by tokenising and searching the result of `getenv("PATH")`.